packages/authenticator/amplify_authenticator/lib/src/utils/validators.dart

```diff
@@ -118,6 +112,26 @@ FormFieldValidator<String> Function(BuildContext) validateNewPassword({
       };
 }
 
+List<PasswordPolicyCharacters> _getUnmetPasswordPolicies(
```

Comment:
Suggestion: Remove use of `PasswordPolicyCharacters`.

`PasswordPolicyCharacters` is part of AmplifyConfig. _getUnmetPasswordPolicies is mapping `PasswordPolicy` (part of AmplifyOutputs) to  `PasswordPolicyCharacters` (part of AmplifyConfig). I don't think we want to map back to AmplifyConfig unless there is a good reason.